urwid/container.py

```diff
@@ -2282,6 +2302,14 @@ def keypress(self, size, key):
 
             self.focus_position = j
             return
+
+        if self._command_map[key] == 'next selectable':
+            # we wanted the next selectable but we were already at the end
+            # set focus back to first selectable
+            # (but still return key to parent in case we should traverse
+            #  to sibling)
+            self.focus_first_selectable()
```

Comment:
changing the focus *and* sending the key back to the parent is a behaviour unlike any other widget. I don't think it makes sense to wrap around inside this container. Maybe let an outer container force wrapping if that's desired (more flexible, allows wrapping through multiple child widgets etc.)